cmd/kubeadm/app/cmd/init.go

```diff
@@ -458,19 +460,16 @@ func (d initData) Tokens() []string {
 
 // runInit executes master node provisioning
 func runInit(i *initData, out io.Writer) error {
```

Comment:
I'm expecting this PR to remove this func after rebasing when all the phases merge